feature_engine/testing/data/cleaned_data/test_chat_level.csv

```diff
@@ -83,4 +83,49 @@ I respond to that too",num_block_quote_responses,2
 5,test3,After the edit he done it made it sound like he really loves his family,certainty_rocklage,6.175
 5,test1,"I don't think the guy is wrong in asking her to learn more english being that she lives in America, but he has to understand she is older and may not have the patience or capacity to learn a lot of english.",certainty_rocklage,5.472
 5,test2,"Learning a second language is easiest when you're a child for a reason. Your brain is wired differently then, which makes it easier.",certainty_rocklage,5.4
-5,test2,I think he tried to help her.� He gave her resources to use and she apparently didn't use them.,certainty_rocklage,4.32
\ No newline at end of file
+5,test2,I think he tried to help her.� He gave her resources to use and she apparently didn't use them.,certainty_rocklage,4.32

```

Comment:
I added in the tests you wrote from the spreadsheet so that the tests will run automatically upon pushing to GitHub :)